elyra/metadata/schemas/component-registry.json

```diff
@@ -38,11 +38,11 @@
           "title": "Category Names",
           "description": "Category names associated with this Component Registry (the components defined in this registry will be organized in the component palette according to these categories)",
           "type": "array",
-          "items": [
-            {
-              "type": "string"
-            }
-          ],
+          "uniqueItems": true,
```

Comment:
This requirement will need to get conveyed to the tooling since this will only be enforced at the time of validation.  Not sure the messaging will be clear from that.

Reply:
You know what, on further inspection I'm thinking of removing this. Looking through how we render the palette information, it looks like I've already accounted for the case of a repeat instance and it won't be a problem.

I was trying to get fancy for no reason 🥲

Reply:
Awesome - thanks for checking on that.  Yeah, this kind of stuff can be more problematic than it's worth.